SpatialGDK/Documentation/content/get-started/starter-template/get-started-template-local.md

```diff
@@ -1,23 +1,32 @@
 <%(TOC)%>
 # The Starter Template
 
-## 2.  Launch a local deployment with multiple clients
+## 2.  Launch a local deployment
 
-When you want to try out your game, you need to run a deployment.
+When you want to try out your game, you need to launch a deployment; the deployment runs the game's functionality. We call the running deployment, the game "simulation" and what's happening in it, the "SpatialOS world".
```

Comment:
Same comment as Ernie's earlier comment about this sentence

Reply:
```suggestion
When you want to try out your game, you need to launch a deployment; the deployment runs the game's functionality. We call the running deployment the game "simulation", and what's happening in it, the "SpatialOS world".
```